mod-circulation/src/main/resources/domain/mod-circulation/features/loans.feature

```diff
@@ -216,3 +217,35 @@ Feature: Loans tests
     Then status 200
     Then match $.loans[0].dueDateChangedByRecall == true
     And match $.loans[0].dueDate !contains expectedDueDateBeforeRequest
+
+  Scenario: When an requested loaned item is checked in at a service point designated as the pickup location of the request, change the item status to awaiting-pickup
+
+    * def extItemBarcode = '' + random(100000)
+    * def extUserId1 = call uuid1
+    * def extUserId2 = call uuid2
+    * def extUserBarcode1 = random(1000000)
+    * def extUserBarcode2 = random(1000000)
+    * def extRequestId = call uuid1
+
+    # post an item
```

Comment:
It looks like we're posting much more than just an item after this comment

Reply:
ok, I will clarify more precisely